rustls/src/msgs/message.rs

```diff
@@ -387,6 +387,7 @@ impl TryFrom<PlainMessage> for Message {
 ///
 /// This type also cannot decode its internals and
 /// cannot be read/encoded; only `OpaqueMessage` can do that.
+#[derive(Debug)]
 pub struct BorrowedPlainMessage<'a> {
```

Comment:
Let's roll the `Debug` derive into b99e527499e96ada274bf4a29383b88b8b27fa22

Reply:
yes, usually a "fixup" commit is for squashing into a previous commit

Reply:
I don't think it was described as a fixup prefixed commit when I left this feedback, but if it was I apologize for missing it!

Reply:
I moved this hunk to the commit you mentioned. Is that what you wanted?